packages/mermaid/src/diagrams/git/gitGraphParserV2.spec.ts

```diff
@@ -1,22 +1,23 @@
 import gitGraphAst from './gitGraphAst.js';
-import { parser } from './parser/gitGraph.jison';
+// @ts-expect-error No types available for JISON
+import { gitGraph as gitGraph } from './gitGraph/gitGraph.jison';
 
 describe('when parsing a gitGraph', function () {
   beforeEach(function () {
-    parser.yy = gitGraphAst;
-    parser.yy.clear();
+    gitGraph.yy = gitGraphAst;
+    gitGraph.yy.clear();
```

Comment:
`parser.yy` is the convention we follow in all the tests. As this is the gitGraph test, parser is fine, as there are no two parsers we might confuse. 
I'd suggest to revert the changes in this file, and keep the changes in this file to a minimum.